modules/plugin/shapefile/src/test/java/org/geotools/data/shapefile/ShapefileDataStoreTest.java

```diff
@@ -82,17 +82,7 @@
 import org.geotools.util.factory.FactoryRegistryException;
 import org.junit.After;
 import org.junit.Test;
-import org.locationtech.jts.geom.Coordinate;
-import org.locationtech.jts.geom.CoordinateXYZM;
-import org.locationtech.jts.geom.Envelope;
-import org.locationtech.jts.geom.Geometry;
-import org.locationtech.jts.geom.GeometryCollection;
-import org.locationtech.jts.geom.GeometryFactory;
-import org.locationtech.jts.geom.LineString;
-import org.locationtech.jts.geom.LinearRing;
-import org.locationtech.jts.geom.MultiLineString;
-import org.locationtech.jts.geom.Point;
-import org.locationtech.jts.geom.Polygon;
+import org.locationtech.jts.geom.*;
```

Comment:
there was a recent PR that made sure all imports were fully qualified, see #3180

Reply:
> would be good to have this backported for 24.2 as well

Generally, I'd suggest that version upgrades should *not* be backported.  In the case of JTS 1.16.0 to 1.17.1, there was an API change which cases problems if downstream users assume that they could go from GT 2.17.0 to 2.17.3 without thinking.  (GeoMesa made this assumption; shame on me!)

That said, I could see it as being useful to backport security fixes as an exception to this rule.

Reply:
Hmm.... is there going to be something in the build to fail the * imports?  (Thanks for catching it; happy to fix it.)